src/Word/DashToSeparator.php

```diff
@@ -34,4 +64,16 @@ private function filterNormalizedValue($value)
     {
         return str_replace('-', $this->separator, $value);
     }
+
+    /** @return $this */
+    public function setSeparator(string $separator): self
```

Comment:
These accessors aren't really needed for this, but removing them would change the interface of this class. Any preference on keeping them or not?